client/apps/city_issues/views.py

```diff
@@ -6,53 +6,73 @@
 from datetime import date, datetime, time
 
 from django.views.generic import CreateView
-from django.views.generic.base import TemplateView
+from django.views.generic.base import TemplateView, View
 from django.contrib import messages
 from django.core import serializers
 from django.http import JsonResponse
 from django.shortcuts import redirect, render
 from django.urls import reverse
 from django.utils.timezone import make_aware
 
-from city_issues.models import Attachments, Issues
-from city_issues.forms.forms import EditIssue, IssueFilter, IssueForm
```

Comment:
Why you delete `IssueFilter`? Now page with map open with error "global name 'IssueFilter' is not defined"